doc-content/enterprise-only/installation/roles-users-con.adoc

```diff
@@ -26,9 +26,4 @@ ifdef::PAM[]
 endif::PAM[]
 
 * `rest-all`: Users with the `rest-all` role can access {CENTRAL} REST capabilities.
-* `kie-server`: Users with the `kie-server` role can access {KIE_SERVER} (KIE Server) REST capabilities. This role is mandatory for users to have access to Manage and Track views in {CENTRAL}.
-
-
-
-
-
+* `kie-server`: Users with the `kie-server` role can access {KIE_SERVER} (KIE Server) REST capabilities. This role is mandatory for users to have access to *Manage and Track* views in {CENTRAL}.
```

Comment:
Ditto about manage and task views, from my previous comment. Also, I think you need to ifdef starting "This role is mandatory ..." because it applies to PAM only, right? You ifdefed this bit for PAM in the other instance where you address this.